<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>568</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Учет времени</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>550</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <widget class="QComboBox" name="cbMonth">
     <property name="geometry">
      <rect>
       <x>427</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Январь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Февраль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Март</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Апрель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Май</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Июнь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Июль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Август</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сентябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Октябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ноябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Декабрь</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="sbYear">
     <property name="geometry">
      <rect>
       <x>427</x>
       <y>47</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2016</number>
     </property>
     <property name="maximum">
      <number>2032</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Сегодня:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdMinutes">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Hex</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Hex</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>1181062</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>62</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Часов:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetTime">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>41</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Время</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tblTime">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>102</y>
      <width>550</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>546</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnTimeCreate">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Создать...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnTimeEdit">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Редактировать...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnTimeDelete">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblBuild">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>541</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>deleteSelectedRow()</slot>
 </slots>
</ui>
